config/Mosaic.py

```diff
@@ -1,3 +1,8 @@
 import os
 from lsst.utils import getPackageDir
 config.astrom.load(os.path.join(getPackageDir("obs_subaru"), "config", "filterMap.py"))
+
+from lsst.pipe.tasks.setConfigFromEups import setPhotocalConfigFromEups
+setPhotocalConfigFromEups(config)
+
+config.load(os.path.join(getPackageDir("obs_subaru"), "config", "srcSchemaMap.py"))
```

Comment:
I've moved things around a bit, but have kept both` Mosaic.py` configs (seeing as we are attempting to maintain the ability to reduce `SuprimeCam` data...if there is a need to adapt `meas_mosaic` to run on such data, this will likely happen as the need is encountered?)  I've pushed a `tickets/DM-2674` branch in `obs_subaru` with all the changes (above and below). Please have a look to see if things are any cleaner in your opinion.

Yes, I moved the `srcSchemaMap` to the `hsc` config level.